Runtime/Scripts/LyumaAv3Runtime.cs

```diff
@@ -434,11 +434,10 @@ public string GetTypeString()
 		};
 		
 		public static readonly Type[] MirrorCloneComponentBlacklist = new Type[] {
-			typeof(Camera), typeof(FlareLayer), typeof(AudioSource), typeof(Rigidbody), typeof(Joint)
+			typeof(Camera), typeof(FlareLayer), typeof(AudioSource), typeof(Rigidbody), typeof(Joint), typeof(IConstraint),
 		};
 		public static readonly Type[] ShadowCloneComponentBlacklist = new Type[] {
-			typeof(Camera), typeof(FlareLayer), typeof(AudioSource), typeof(Light), typeof(ParticleSystemRenderer), typeof(Rigidbody), typeof(Joint)
-		
+			typeof(Camera), typeof(FlareLayer), typeof(AudioSource), typeof(Light), typeof(ParticleSystemRenderer), typeof(Rigidbody), typeof(Joint), typeof(IConstraint),
```

Comment:
I don't know much about shadow clone bit I assumed it's similar to mirror clone so I added constraint to this blacklist. is this correct?